components/menu/themes/spectrum.css

```
@@ -17,7 +17,7 @@
 
 @container style(--system: legacy) {
 	.spectrum-Menu {
-		--spectrum-menu-item-background-color-default: var(--spectrum-gray-50);
+		--spectrum-menu-item-background-color-default: var(--spectrum-gray-100);
```

I don't think I see a change visually now when a menu item is hovered. Should I be expecting one?

---

The styles for `--spectrum-menu-item-background-color-default` on the current S1 branch are actually [transparent](--spectrum-menu-item-background-color-default). I wonder if we should be correcting this to be:
 
```suggestion
		--spectrum-menu-item-background-color-default: var(--spectrum-transparent-black-100);
```